app/routes.py

```diff
@@ -1 +1,215 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, abort, make_response, request
+import requests
+import os
+from sqlalchemy import text
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from .routes_helpers import validate_model
+from datetime import datetime
+
+bp = Blueprint("tasks", __name__, url_prefix="/tasks")
```

Comment:
Consider calling this `Blueprint` `task_bp` in order to keep it consistent like your second `Blueprint`.